helpers/dbload.ts

```diff
@@ -96,6 +99,34 @@ UserLesson.belongsTo(User, {
 
 sequelize.sync({ alter: !!process.env.ALTER_DB })
 
+/* istanbul ignore next */
+if (process.env.NODE_ENV === 'development') {
+  //add data after fresh install
+  Lesson.findAll({
+    limit: 1
+  }).then((lessons: Array<LesssonTypescriptTypes>) => {
+    if (!lessons[0]) {
+      Lesson.bulkCreate(lessonData)
+      console.log('Successfully inserted lesson data')
+    }
+  })
+
+  Challenge.findAll({
+    limit: 1
+  }).then((challenges: Array<ChallengeTypescriptTypes>) => {
+    if (!challenges[0]) {
+      lessonData.map(l => {
+        //need to add lessonID to comply with Challenge schema
+        let validChalleges: Array<ChallengeTypescriptTypes> = []
```

Comment:
should be `validChallenges` instead of `validChalleges`,
also needs to be declared `const` since its value does not change